src/components/shop-team-data.vue

```diff
@@ -27,10 +27,15 @@
 </template>
 
 <script>
+import itemDisplay from  '../components/item-display.vue'
+
 export default {
   name: 'team-data',
```

Comment:
Name should be capitalized to TeamData.
Filenames need to be camel-case with no dashes in them even though I made the first filename like that. This isn't me being picky, some of the vue component parts won't work unless it's named that way.